source/main.c

```diff
@@ -355,6 +364,17 @@ int main()
         PAD_ButtonsHeld(PAD_CHAN3)
     );
 
+    if (all_buttons_held & PAD_BUTTON_LEFT || SYS_ResetButtonDown())
+    {
+        // Since we've disabled the Qoob, we wil reboot to the Nintendo IPL
+        kprintf("Skipped. Rebooting into original IPL...\n");
+        delay_exit();
+        return 0;
+    }
+
+    char *paths[2];
+    int num_paths = 0;
+
```

Comment:
Ah right, this is what slipped in heh. I'm cherry-picking your changes manually so it's fine.